### ISSUE TITLE
Images!

### ISSUE THREAD

**Issue by flowirtz** [openclimatefix/website#1]
Apart from the logos there is currently not a single picture on the website.
We should at least add an image to every single blogpost, even if it's stock footage.
To integrate with Medium we'd need to have one anyways.

We should also look in what other places we can put images as well. 😊 

**ToDos**
- [x] Research good CC0 images to use
- [x] Update `gatsby-node.js` to support images
- [x] Add cover images to all blogs posts in `/src/posts/*`

**Comment by JackKelly** [comment]
Totally agree with this!  Now that I've got the satellite imagery into a form that I can actually use (!), I can create some figures / photos from satellite data.  And, as my research progresses, I should have more visual results to share (like the videos of cloud cover and PV).

Speaking of which... Should we embed [the video of cloud cover and PV](https://www.youtube.com/watch?v=6Wz8KajCKgQ) on the website?  Perhaps on the Projects page?
